### PR TITLE
Update Exosphere volume mounting docs

### PR DESCRIPTION
**Pull request by kageurufu**
Companion PR to https://gitlab.com/exosphere/exosphere/-/merge_requests/892

When merged, Exosphere will mount volumes to /media/volume/[volume_name] automatically. This PR
 changes the documentation to reflect that.

### CODE REVIEW

File: docs/ui/exo/storage.md [hunk 1]
```diff
@@ -23,11 +23,13 @@ Volumes can be attached from either the *Volumes* section of the Instance Detail
...
+After successfully attaching a volume to an instance, Exosphere will display the mount point (`/media/volume/my-volume`) to which the volume is attached. This information can also be found on your Instance Details page or Volume Details page.
 
 The mount point is the directory that represents the root of a volume's filesystem and is where you can access files stored in it.
 
-Please note that attached volumes are mounted the first time they are accessed inside the instance (e.g. with a `cd` command), and are NOT necessarily mounted at attachment time. If you encounter a seeming lag or problem with filesystem access, the volume may not be properly mounted; you may wish to try making sure you can `cd` to the directory.
+#### Note for older Instances
+
+On older Instances (created before 2/10/2024), the device location (`/dev/sdb`) will also be displayed, and the mount point will match the device location (`/media/volume/sdb`).
```
Comment (c-mart) [inline docs/ui/exo/storage.md:32]: N.B. to whoever merges, update this date if needed.

**Review by c-mart — APPROVED** [review]
Looks good!

Let's merge this at the same time https://gitlab.com/exosphere/exosphere/-/merge_requests/892 is merged (currently waiting @julianpistorius review).

File: docs/ui/exo/storage.md (hunk 1 above)
Comment (julianpistorius) [inline docs/ui/exo/storage.md:32]:
```suggestion
On older Instances (created before 2024-02-08), the device location (`/dev/sdb`) will also be displayed, and the mount point will match the device location (`/media/volume/sdb`).
```

**Comment by julianpistorius** [comment]
Thank you @kageurufu!

**Pull request merged**